### PR TITLE
added README file with instructions for essaycheck_script.py file

### PR DESCRIPTION
**Pull request by cryptobellum**
# Description

I added a README.md file to explain what the essaycheck_script.py file does and how to run it.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [X] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: utils/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+**Explanation of essaycheck_script.py**
+
+**How to Run This Script:**
+
+Place the script in the repo's top-level directory. cd into the repo from the command line and enter the following:
+```angular2html
```
Comment (singparvi) [inline utils/README.md:6]: Although this serves the purpose you should have used bat or bash in my opinion. More information here --> [link](https://support.codebasehq.com/articles/tips-tricks/syntax-highlighting-in-markdown)

**Comment by singparvi** [comment]
Nice work Sharon. This README is nicely written. I have made a minor suggestion.

**Comment by singparvi** [comment]
Nice work Sharon. This README is nicely written. I have made a minor suggestion.

**Review by ike-okonkwo — APPROVED** [review]
Thanks for updating the documentation

**Review by singparvi — APPROVED** [review]
Nice work. Approved.

**Comment by cryptobellum** [comment]
> Thanks for updating the documentation

I'm not sure how this merge will be handled now. I submitted my PR before Parvi's recent PR, and he used my branch, so his PR had two commits, and one of those commits was this README.

**Pull request merged**